cli/cli.go

```diff
@@ -6,14 +6,14 @@ import (
 
 	log "github.com/sirupsen/logrus"
 	"k8s.io/klog/v2"
-
-	"github.com/argoproj/pkg/errors"
 )
 
 // SetLogLevel parses and sets a logrus log level
 func SetLogLevel(logLevel string) {
 	level, err := log.ParseLevel(logLevel)
-	errors.CheckError(err)
```

Comment:
Do you want to delete the `CheckError` function and the internal `errors` package entirely? I suppose that would be a breaking change worth separating out though